s4e-mac.core_desc

```diff
@@ -1,6 +1,6 @@
-import "CoreDSL-Instruction-Set-Description/RISCVBase.core_desc"
+import "CoreDSL-Instruction-Set-Description/RV32I.core_desc"
```

Comment:
As justified in 28.3.24 meeting, is sensible that this MAC extension only offers benefit to 32-bit core